modules/appeals_api/config/locales/en.yml

```diff
@@ -5,6 +5,7 @@ en:
         hearing_type_preference_inclusion: "If '/data/attributes/boardReviewOption' 'direct_review' or 'evidence_submission' is selected, '/data/attributes/hearingTypePreference' must not be selected"
         hearing_type_preference_missing: "If '/data/attributes/boardReviewOption' 'hearing' is selected, '/data/attributes/hearingTypePreference' must also be present"
         mismatched_ssns: "Request header 'X-VA-SSN' does not match the associated appeal's SSN"
+        mismatched_ssns_in_body: "The provided 'ssn' value does not match the associated appeal's SSN"
```

Comment:
This is the same error as in decision reviews, but since it's no longer in the header I added a different error message here